src/test/java/co/rsk/bitcoinj/script/ScriptTest.java

```diff
@@ -632,26 +631,55 @@ public void getSigInsertionIndex_whenP2shScript_shouldReturnZero() {
         Sha256Hash hashForSignature = Sha256Hash.of(new byte[]{1});
         BtcECKey signingKey = BtcECKey.fromPrivate(BigInteger.valueOf(800));
 
-        int actualSigInsertionIndex = p2shOutputScript.getSigInsertionIndex(hashForSignature, signingKey);
-        Assert.assertEquals(EXPECTED_DEFAULT_SIG_INSERTION_INDEX_FOR_NO_REDEEM_SCRIPT, actualSigInsertionIndex);
+        assertThrows(NullPointerException.class, () -> p2shOutputScript.getSigInsertionIndex(hashForSignature, signingKey));
```

Comment:
We need to update the name of the test now

Reply:
Done